book/translation.rst

```diff
@@ -428,6 +428,14 @@ via the ``request`` object::
 
         $request->setLocale('en_US');
     }
+    
+.. note::
+
+    Setting the locale using the ``$request->setLocale()`` method won't affect 
+    rendering in the same action. The translator locale is set during the
+    kernel.request event. Either set the locale before the listener is called
```

Comment:
sorry, one more thing before I'll label it as "finished": Could you please put `kernel.request` in a literal (double backticks) as well?